src/main/java/com/gentics/graphqlfilter/filter/operation/Combiner.java

```diff
@@ -0,0 +1,47 @@
+package com.gentics.graphqlfilter.filter.operation;
```

Comment:
This whole package has to be somehow moved around or split, to be accessible for non-GQL related Mesh libraries.